Language/Haskell/GhcMod/Logging.hs

```diff
@@ -45,6 +45,11 @@ gmSetLogLevel :: GmLog m => GmLogLevel -> m ()
 gmSetLogLevel level =
     gmlJournal $ GhcModLog (Just level) (Last Nothing) []
 
+gmGetLogLevel :: forall m. GmLog m => m GmLogLevel 
+gmGetLogLevel = do
+  GhcModLog { gmLogLevel = Just level } <-  gmlHistory
```

Comment:
`fromJust . gmLogLevel <$> gmlHistory`

Thought I'm not sure why you added this function since it's not used anywhere?